core/upgrade/upgrades.php

```diff
@@ -885,4 +911,64 @@ private static function notice( $message ) {
 		$logger = Plugin::$instance->logger->get_logger();
 		$logger->notice( $message );
 	}
+
+	/**
+	 * @param \wpdb $wpdb
+	 * @param string $element_type
+	 *
+	 * @return array
+	 */
+	public static function get_post_ids_by_element_type( $updater, string $element_type ): array {
+		global $wpdb;
+
+		return $updater->query_col(
+			'SELECT `post_id`
+					FROM `' . $wpdb->postmeta . '`
+					WHERE `meta_key` = "_elementor_data" 
+					AND `meta_value` LIKE \'%"elType":"' . $element_type . '"%\';'
+		);
+	}
+
+	/**
+	 * @param $data
+	 *
+	 * @return array|mixed
+	 */
+	private static function convert_to_inner_containers( $data ) {
```

Comment:
```suggestion
	private static function maybe_convert_to_inner_containers( $data ) {
```
@TzviRabinovitch Also here?